Makefile

```diff
@@ -189,7 +189,7 @@ test:
 		grep -v '^\./tpm/aziot-tpm-sys/azure-iot-hsm-c/' | \
 		grep -v '\.generated\.rs$$' | \
 		while read -r f; do \
-			if [ "$$(tail -c 1 "$$f" | wc -l)" -eq '0' ]; then \
+			if [[ -s "$$f" && "$$(tail -c 1 "$$f" | wc -l)" -eq '0' ]]; then \
```

Comment:
Instead of this, just do `grep -v '^\.aziot/test-files/'`

Reply:
I feel like my solution is a bit more robust, since if we ever add an empty-file elsewhere in the repo, things will continue to work okay. That said, I'm not dead-set on leaving it in, so I can change it if you like.

Reply:
If we add an empty file, it should have a copyright header unless we decide otherwise.

Reply:
This doesn't have anything to do with copyright. This is only for the newline check

Reply:
Derp. Sorry. In that case keep your original change.